adapted postgresql-role to CentOS 7 and fixed tiny $PATH issue in rbenv-role

tnt opened the pull request:
As announced...

arBmind commented:
Thanks a lot.

I left some comments on your changes. I hope you can give it some cleanup. I am looking forward to merge this.

tnt commented:
Oh! All comments are gone! This is obviously a side effect of replacing a commit completely by a new one. - Yet another new experience. Not really desirable.

I removed the comment-rubbish (sorry for the mess) and fixed a little mistake you made me aware of with your question regarding the location of the pids/sockets.

The socket is by default in `/tmp` on RedHat but there is no default external pidfile. I had considered the socket `/tmp/.s.PGSQL.5432` as the pidfile (and such probably caused pg to write its pid to its own socket) but fixed this now.

arBmind commented:
Take this link if you want to see the comments: https://github.com/tnt/ansible-rails/commit/c1ef18a21af65bf927bfb84ead12349ccbe21851
